operator/controller/pom.xml

```diff
@@ -89,10 +57,30 @@
       <artifactId>lombok</artifactId>
       <scope>provided</scope>
     </dependency>
+
   </dependencies>
 
   <build>
+
+    <resources>
```

Comment:
with all the properties in `application.properties` this section should not be needed.

Reply:
It is needed to enable filtering.

Reply:
why is filtering necessary?

Reply:
https://github.com/Apicurio/apicurio-registry/pull/5465/files#diff-1eb4a86a818066616518819a8239b71789f7fae867b1d385c999bfe222dd3cafR55-R56
https://github.com/Apicurio/apicurio-registry/pull/5465/files#diff-1eb4a86a818066616518819a8239b71789f7fae867b1d385c999bfe222dd3cafR67

Reply:
oh, ok, can you add a comment to make this explicit?

Reply:
Done.